mcp/src/main/java/io/modelcontextprotocol/client/McpSyncClient.java

```diff
@@ -71,7 +71,7 @@ public class McpSyncClient implements AutoCloseable {
 	 */
 	@Deprecated
```

Comment:
Looks like this deprecation annotation is incorrect

Reply:
It is correct. The intention was to allow us to hide the visibility in 0.9.0. We can also remove the deprecated javadoc note above now.